app/features/build/build_websocket_backend.rb

```diff
@@ -75,9 +65,9 @@ def call(env)
       ws.on(:close) do |event|
         logger.debug([:close, ws.object_id, event.code, event.reason])
 
-        url_details = fetch_build_details(event)
-        build_number = url_details[:build_number]
-        project_id = url_details[:project_id]
+        params = Rack::Request.new(env).params
```

Comment:
`params` isn't very descriptive, let's do `request_params` since `params` is overloaded in ruby.

Reply:
fixed